php/shell.nix

```diff
@@ -0,0 +1,6 @@
+{ pkgs ? import <nixpkgs> {} } : pkgs.mkShell {
+	buildInputs = with pkgs; [
+		php81
+		php81Packages.php-cs-fixer
```

Comment:
I was using `composer` to install php-cs-fixer because I found that the OS-installed version of php-cs-fixer was old and gave different results on my laptop vs github actions.

I think we probably want a script similar to py_env.sh which detects if vendor packages exists, and if not, uses composer to install them?